azure-pipelines.yml

```
@@ -29,34 +33,137 @@ resources:
-# report error
-- template: azure-pipelines-templates/discord-webhook-task.yml@templates  
-  parameters:
-    status: 'failure'
-    webhookUrl: '$(DiscordWebhook)'
-    message: ''
+    value: true
+
+jobs:
+
+##############################
+- job: Build_Library
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 0

(indentation)